cs/planet.go

```diff
@@ -469,15 +471,17 @@ func computePlanetSpec(rules *Rules, player *Player, planet *Planet) PlanetSpec
 	if race.Spec.InnateScanner {
 		spec.Scanner = "Organic"
 		spec.ScanRange = int(float64(planet.innateScanner(player, productivePop)) * player.Race.Spec.ScanRangeFactor)
+		hull := player.Designs[spec.PlanetStarbaseSpec.StarbaseDesignNum - 1].Hull
+		if !player.Race.Spec.NoAdvancedScanners && (hull == "Ultra Station" || hull == "Death Star") {
```

Comment:
Oh whoops, the Ultra Station and Death Star actually have an `InnateScanRangePenFactor` field, I just never put it here by mistake. Rather than checking for the names, can you use that value on the tech?

https://github.com/sirgwain/craig-stars/blob/develop/cs/techstore.go#L2524

We'll need to add it to shipdesign computeScanRanges (https://github.com/sirgwain/craig-stars/blob/develop/cs/shipdesign.go#L493)

Since the staircase is doing the scanning anyway, we can populate the planet ScanRange/ScanRangePen using the starbase values.

Reply:
https://github.com/sirgwain/craig-stars/pull/26

I fixe the AR Pen Scan thing in this PR. Can you revert that piece of your PR please? I'll merge the PR after that. Thanks again for the help!